app/assets/stylesheets/theme/custom.css.scss

```diff
@@ -41,4 +41,42 @@ select.form-control.search_options {
     padding: 5px;
     width: 138px;
   }
+}
+
+// pagination
+.pagination a, .pagination span.current, .pagination span.gap {
+  float: left;
```

Comment:
Properties should be ordered background-color, border-left-width, float, line-height, padding, text-decoration<br>Each selector in a comma sequence should be on its own line